lib/widgets/action_sheet.dart

```diff
@@ -43,18 +44,19 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
   }) : assert(messageListContext.findAncestorWidgetOfExactType<MessageListPage>() != null);
 
   IconData get icon;
-  String get label;
+  String Function(ZulipLocalizations) get translateLabel;
```

Comment:
Separately, it doesn't seem like this benefits from being a getter, now that it's returning a function — it should just be a method, now taking an argument but still returning a `String`.